README.rst

```diff
@@ -193,39 +193,43 @@ Features
     input code      full name
 
       ar              Arabic
-      ru              Russian
-      nl              Dutch
+      be              Belarusian
+      bg              Bulgarian
+      da              Danish
       de              German
+      el              Greek
       en              English
       es              Spanish
+      et              Estonian
+      fa              Persian
+      fi              Finnish
       fr              French
       he              Hebrew
+      hi              Hindi
```

Comment:
Please delete this, you are only adding Thai, there are three new languages accidentally introduced which don't have stopword support